include/wx/univ/textctrl.h

```diff
@@ -451,6 +451,12 @@ class WXDLLIMPEXP_CORE wxTextCtrl : public wxTextCtrlBase,
     bool DoCut();
     bool DoPaste();
 
+#ifdef __WXMSW__
+public:
+    // override MSWHandleMessage to process WM_GETDLGCODE
+    bool MSWHandleMessage(WXLRESULT *result, WXUINT message, WXWPARAM wParam, WXLPARAM lParam);
```

Comment:
If we really need to override it, it should use `override`:
```suggestion
    bool MSWHandleMessage(WXLRESULT *result, WXUINT message, WXWPARAM wParam, WXLPARAM lParam) wxOVERRIDE;
```